public/scripts/worktrollyUI.js

```diff
@@ -61,10 +79,23 @@ $("#submitCreateNewTask").click(function() {
     console.log(category);
     console.log(createNewTaskSprintNumber);
 
-    var createNewTaskFunction = firebase.functions().httpsCallable('createNewTask');
-    createNewTaskFunction({ Title: title, Description: des, Priority: priority, Difficulty: difficulty, Creator: creator, Assignee: assignee, EstimatedTime: estimatedTime, Status: status, Category: category, CreateNewTaskSprintNumber: createNewTaskSprintNumber }).then(result => {
+    var createNewTaskFunction = firebase
+        .functions()
+        .httpsCallable("createNewTask");
```

Comment:
Indentation issue please talk to shivam for js formatter extension, and fix them